donate/thunderbird/templates/payment/thank_you.html

```diff
@@ -10,21 +10,23 @@
 {% block buttons %}
     {% trans "I donated to @mozthunderbird today to #freetheinbox. Join me to support communication privacy." as twitter_text context "Used as a tweet" %}
     {% trans "I donated to Thunderbird today" as email_subject context "Email subject line" %}
+    {% get_current_language as LANGUAGE_CODE %}
+    {% with page_link="https://donate.mozilla.org/"|add:LANGUAGE_CODE|add:"/thunderbird/"%}
 
-    <a class="button button--outline button--icon" href="https://www.facebook.com/sharer/sharer.php?u=https://donate.mozilla.org/{% get_locale %}/thunderbird/" title="{% trans 'Share on Facebook' %}">
+    <a class="button button--outline button--icon" href="https://www.facebook.com/sharer/sharer.php?u={{ page_link }}" title="{% trans 'Share on Facebook' %}">
         <svg class="button__icon button__icon--facebook" width="16" height="16">
             <use xlink:href="#facebook"></use>
         </svg>
         <div class="button__label">{% trans "Facebook" %}</div>
     </a>
     {# see https://dev.twitter.com/web/tweet-button/web-intent #}
-    <a class="button button--outline button--icon" href="https://twitter.com/intent/tweet?text={{ twitter_text|urlencode }}&amp;url=https://donate.mozilla.org/{% get_locale %}/thunderbird/&amp;via=mozthunderbird" title="{% trans 'Share on Twitter' %}">
+    <a class="button button--outline button--icon" href="https://twitter.com/intent/tweet?text={{ twitter_text|urlencode }}&amp;url={{ page_link }}&amp;via=mozthunderbird" title="{% trans 'Share on Twitter' %}">
```

Comment:
are we missing a question mark here before `&amp;`??? And do we even need `&amp;`?

Reply:
We already have a question mark earlier in the URL `https://twitter.com/intent/tweet?`, but you’re absolutely right that we can replace the `&amp;` with `&` for all these links

Reply:
Oh yea true. I totally didn't see the `?` earlier in the URL haha